third_party/cabal2bazel/bzl/cabal_package.bzl

```diff
@@ -189,7 +109,25 @@ _cabal_haskell_macros = rule(
 def _paths_module(desc):
   return "Paths_" + desc.package.pkgName.replace("-","_")
 
-def _glob_modules(src_dir, extension):
+def _hazel_symlink_impl(ctx):
+  ctx.actions.run(
+      outputs=[ctx.outputs.out],
+      inputs=[ctx.file.src],
+      executable="ln",
```

Comment:
This suggests there's no generic abstraction in bazel to deal with symlinking. Oh well.

Reply:
It seems like it at this point; see in particular https://github.com/bazelbuild/bazel/issues/4681